charts/zitadel/templates/deployment_login.yaml

```diff
@@ -0,0 +1,119 @@
+{{- if .Values.login.enabled }}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "zitadel.login.fullname" . }}
+  labels:
+    {{- include "zitadel.login.labels" . | nindent 4 }}
+  {{- with .Values.login.annotations }}
+  annotations:
+    {{- toYaml . | nindent 4 }}
+  {{- end }}
+spec:
+  replicas: {{ .Values.replicaCount }}
+  selector:
+    matchLabels:
+      {{- include "zitadel.login.labels" . | nindent 6 }}
+  template:
+    metadata:
+      annotations:
+        {{- with .Values.login.podAnnotations }}
+          {{- toYaml . | nindent 8 }}
+        {{- end }}
+        checksum/configmap: {{ include (print $.Template.BasePath "/configmap_login.yaml") . | sha256sum }}
+      labels:
+        {{- include "zitadel.login.labels" . | nindent 8 }}
+        {{- with .Values.podAdditionalLabels }}
+          {{- toYaml . | nindent 8 }}
+        {{- end }}
+    spec:
+      {{- with .Values.imagePullSecrets }}
+      imagePullSecrets:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      serviceAccountName: {{ include "login.serviceAccountName" . }}
+      securityContext:
+        {{- toYaml .Values.login.podSecurityContext | nindent 8 }}
+      enableServiceLinks: false
+      containers:
+      {{- if .Values.login.extraContainers }}
+        {{- toYaml .Values.login.extraContainers | nindent 8 }}
+      {{- end }}
+        - name: {{ .Chart.Name }}-login
+          securityContext:
+            {{- toYaml .Values.login.securityContext | nindent 14 }}
+          image: "{{ .Values.login.image.repository }}:{{ .Values.login.image.tag | default .Chart.AppVersion }}"
+          imagePullPolicy: {{ .Values.login.image.pullPolicy }}
+          env:
+            - name: NEXT_PUBLIC_BASE_PATH
+              value: /ui/v2/login
+          ports:
+          - containerPort: 3000
```

Comment:
Does this prot not have to be configurable as well?

Reply:
I centralized it :+1: